lib/EquivValid.thy

```diff
@@ -575,7 +575,7 @@ lemmas pre_ev =
 subsection\<open>Tom instantiates wpc\<close>
```

Comment:
I know this isn't yours, but this section title ...

Reply:
Ha, I did not see that :-) I should probably adjust this.